scripts/run.sh

```diff
@@ -49,6 +49,9 @@ fi
 if [[ -z $serve_tiles ]]; then
   serve_tiles="True"
 fi
+if [[ "{$force_rebuild}" == "True" ]]; then
+  build_tar="Force"
+fi
```

Comment:
not sure that's enough, see https://github.com/gis-ops/docker-valhalla/issues/135#issuecomment-1908511375. maybe it's best to export the `do_build` in the other shell script and use that in the build_tar function further up?

Reply:
That would be ideal, but the way we are sourcing that `configure_valhalla.sh` script does not export variables to the sourcing context... the way we are sourcing this is getting a bit messy anyway, this would make it even more intransparent imo. For now I set the `--overwrite` flag on `valhalla_build_extract` which surely is to blame for previous unexpected behavior.